### PR TITLE
Fix leaking test to stdout

### PR DESCRIPTION
**Pull request by seuros**

### CODE REVIEW

File: lib/trailblazer/developer/wtf.rb [hunk 1]
```diff
@@ -5,7 +5,9 @@ def wtf(activity, *args)
     Wtf.invoke(activity, *args)
   end
 
-  singleton_class.alias_method :wtf?, :wtf
```
Comment (seuros) [inline lib/trailblazer/developer/wtf.rb:8]: singleton_class.alias_method  breaks in ruby bellow 2.5

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,9 @@
 sudo: false
 language: ruby
 rvm:
-  - 2.3.1
-before_install:
-  - gem install bundler
+  - 2.6.2
+  - 2.5.5
+  - 2.4.4
+  - 2.3.7
```
Comment (emaglio) [inline .travis.yml]: since we are supporting ruby version >= 2.1.0 I suggest to add 2.2.x and 2.1.x here

**Review by emaglio — DISMISSED** [review]
Looks good to me - what is `gem.local.rb` for?

File: .travis.yml (hunk 2 above)
Comment (emaglio) [reply .travis.yml]: Let's move these version under allow failure like here: https://github.com/trailblazer/trailblazer-activity-dsl-linear/pull/3

What do you think?

**Review by emaglio — APPROVED** [review]
:+1:

**Comment by seuros** [comment]
@emaglio `gem.local.rb` references to gems locally.  
You can use it by setting `BUNDLE_GEMFILE=gem.local.rb`

**Pull request merged**